spec/controllers/to_dos_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe ToDosController, type: :controller do
+
+    let(:to_do) { FactoryGirl.create(:to_do) }
+    let(:json) { JSON.parse(response.body) }
+
+    describe "GET /to_dos.json" do
+      before do
+        get :index
+      end
+      it "returns all to_dos in json format" do
+        expect(json).to include
+          ({
+            "title": "I have to do this thang",
+            "is_completed": true
+            })
+      end
+    end
+
+    describe "POST /to_dos.json" do
+      before do
+        post :create, to_do: { title: "New title", is_completed: true }
+      end
+      it "creates a new to_do" do
+        expect(ToDo.last.title).to eq "New title"
+        expect(json["to_do"]["title"]).to eq "New title"
+        expect(json["to_do"]["is_completed"]).to eq true
```

Comment:
Same thing here, actually; sorry for going a bit out of order. You can perhaps assert that the number of Todos in the DB changed by +1.